backends-common/postgres/src/main/java/org/apache/james/backends/postgres/utils/DomainImplPostgresConnectionFactory.java

```diff
@@ -31,6 +31,7 @@
 
 import io.r2dbc.spi.Connection;
 import io.r2dbc.spi.ConnectionFactory;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 public class DomainImplPostgresConnectionFactory implements JamesPostgresConnectionFactory {
```

Comment:
Q: As I understand `PoolBackedPostgresConnectionFactory`  supports rls for domains. Why do we still need `DomainImplPostgresConnectionFactory`?

Reply:
Does the support configurable for enable/disable pool make sense?

Reply:
I tend to NO, pick one that best

Reply:
Pool with a maximum of 1 connection configured is likely no pool already I think.

Reply:
Anything to do regarding this?

Reply:
I would like to run performance test with pool for jmap and rls before making decision on this DomainImplPostgresConnectionFactory